ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/defs_interface_ip_nat_static.schema.yml

```diff
@@ -35,6 +34,7 @@ $defs:
           description: IPv4 address.
         original_port:
           type: int
+          description: The combination of original_ip and original_port must be unique.
```

Comment:
Maybe also add this for the original_ip

Reply:
Done. Added to translated_ip and translated_port too.